node/src/accountant/db_access_objects/failed_payable_dao.rs

```diff
@@ -68,7 +75,10 @@ pub trait FailedPayableDao {
     fn get_tx_identifiers(&self, hashes: &HashSet<TxHash>) -> TxIdentifiers;
     fn insert_new_records(&self, txs: &[FailedTx]) -> Result<(), FailedPayableDaoError>;
     fn retrieve_txs(&self, condition: Option<FailureRetrieveCondition>) -> Vec<FailedTx>;
-    fn update_tx_failures(&self, hash_map: &FailureUpdates) -> Result<(), FailedPayableDaoError>;
+    fn update_recheck_status(
```

Comment:
This should probably be either "update_status_rechecked" or simply "mark_as_rechecked".